examples/cpp/SampleCpp/main.cpp

```diff
@@ -387,6 +388,8 @@ int main()
     // normal	- 4 sec
     // low		- 8 sec
 
+    printf("LogManager::Flush\n");
```

Comment:
Is this `printf` intended?

Reply:
I was just matching what was there for `    printf("LogManager::FlushAndTeardown\n");` I can remove it if you'd like